4-fleetscope/envs/development/main.tf

```diff
@@ -17,7 +17,11 @@
 locals {
   env = "development"
 
-  namespace_ids = ["frontend", "accounts", "transactions"]
+  namespace_ids = {
```

Comment:
Rather than hardcode namespaces in the actual TF, I recommend making this a variable and putting these values in a `terraform.tfvars` similar to: https://github.com/GoogleCloudPlatform/terraform-google-enterprise-application/blob/main/2-multitenant/terraform.tfvars

Reply:
hmmm, the team groups are created dynamically during setup phase, they can not be hard coded in tfvars

Reply:
discussed offline, passed the whole namespace_ids through variable injected from integration test